src/services/applications/applicantService.ts

```diff
@@ -123,6 +124,22 @@ export class ApplicantService implements ApplicantServiceInterface {
     return result;
   };
 
+  public deleteCV = async (fileName: string): Promise<string> => {
+    if (!process.env.DROPBOX_API_TOKEN) {
+      throw new Error("Failed to delete CV from Dropbox, set dropbox envs correctly");
+    }
+    if (!process.env.DROPBOX_API_TOKEN) throw new Error("Failed to delete CV from Dropbox, set dropbox envs correctly");
```

Comment:
```suggestion
```

Looks like it's just a duplicate of the check above.

Reply:
Sorry about that, it was a duplicate. I just removed it.